content/articles/an-introduction-to-defi/index.md

```diff
@@ -0,0 +1,129 @@
+
+#### The Asset Layer
+The asset layer implements the different financial assets, such as the native protocol asset (Ethereum). In this layer, there are standard protocols on how to interact with the token contract. ERC20 is one of the standards used on this layer to maintain a different balance for the asset. 
+
+#### The Protocol Layer
+The protocols are created as smart contracts with rules defined to govern tasks and activities on the asset. The protocol layer is where different types of financial services are implemented such as Lending, Derivatives, Asset Management, and more. As such, it provides liquidity to the DeFi ecosystem.
+
+#### The Application Layer
+The application layer is user-facing. It is an intermediary between the user and the cloud servers that interpolate with the entire stacks of the blockchain and particularly, the underlying protocol layer.
+
+#### The Aggregation Layer
+This layer consists of different applications from the application layer all stacked up into one application to provide user access to other DeFi services. An example is a cryptocurrency wallet with multiple services such as staking, lending and exchange.
+
+### DeFI Building Block and Services
+The following are some of the building blocks and services of Decentralized Finance:
+- Asset Tokenization.
+- Stablecoin.
+- Decentralized Exchanges. 
+
+#### Asset Tokenization
+Tokenization is the process of securitizing an asset into the blockchain network. Upon tokenization of such security, the representation of the asset on the blockchain is what we call “Tokens”. 
+ 
+Asset tokenization can be in the form of a Governance Token whereby the future dimensioned pathway of the token is decided by a vote on the blockchain. A security token is also another form of asset tokenization whereby the token is used to represent the state of a real-life asset.
+
+#### Stablecoin
+These are tokens whose price is tagged to the type of assets such as fiat and precious metal. Stable coins can be created via Off-chain Collateral(Fiat), OnChain Collateral (Crypto Asset), Non-Collateralized (Algorithmic).
+
+#### Decentralized Exchanges
+Decentralized exchange is also another service of DeFi. It is a marketplace that allows users to trade different assets in the financial market. They are dimensioned across custodial and non-transparent principality which are essentially Centralized Exchanges and non-custodial and transparency for decentralized exchanges.
+
+### Risk and Security Structures in DeFi
+The entire blockchain network is susceptible to security breaches of all types. This article is beyond the scope of examining all the types of security risks in DeFi but as a pointer on the risks that exist on all layers of DeFi, they include:
+
+- Network Attacks such as Eclipse/DOS Attack
+- Consensus Attacks
+- Smart Contract Code Bugs
+- DeFi Protocol Composability attack
+- Bridge Attacks and Governance Attacks
```

Comment:
Since you won't be explaining them, it will be helpful to add external links to each of the types of security.